LICENSE.md

```diff
@@ -0,0 +1,55 @@
+# Blue Oak Model License
```

Comment:
interesting licence, where did you get it from ?

Reply:
😆  came from the template generation